cmd/notation/common.go

```diff
@@ -31,27 +31,36 @@ var (
 		fs.StringVarP(p, flagPassword.Name, flagPassword.Shorthand, "", flagPassword.Usage)
 	}
 
-	flagPlainHTTP = &pflag.Flag{
-		Name:     "plain-http",
-		Usage:    "registry access via plain HTTP",
+	flagInsecureRegistry = &pflag.Flag{
+		Name:     "insecure-registry",
+		Usage:    "use HTTP protocol while connecting to registries. Use it only for testing purposes. (alias: --plain-http)",
 		DefValue: "false",
 	}
-	setFlagPlainHTTP = func(fs *pflag.FlagSet, p *bool) {
-		fs.BoolVar(p, flagPlainHTTP.Name, false, flagPlainHTTP.Usage)
+	setFlagInsecureRegistry = func(fs *pflag.FlagSet, p *bool) {
+		fs.BoolVar(p, flagInsecureRegistry.Name, false, flagInsecureRegistry.Usage)
 	}
 )
 
 type SecureFlagOpts struct {
-	Username  string
-	Password  string
-	PlainHTTP bool
+	Username         string
+	Password         string
+	InsecureRegistry bool
 }
 
 // ApplyFlags set flags and their default values for the FlagSet
 func (opts *SecureFlagOpts) ApplyFlags(fs *pflag.FlagSet) {
 	setflagUsername(fs, &opts.Username)
 	setFlagPassword(fs, &opts.Password)
-	setFlagPlainHTTP(fs, &opts.PlainHTTP)
+	setFlagInsecureRegistry(fs, &opts.InsecureRegistry)
+	fs.SetNormalizeFunc(aliasNormalizeFunc)
 	opts.Username = os.Getenv(defaultUsernameEnv)
 	opts.Password = os.Getenv(defaultPasswordEnv)
 }
+
+func aliasNormalizeFunc(f *pflag.FlagSet, name string) pflag.NormalizedName {
```

Comment:
Recommend removing aliasing to avoid breaking post RC to GA. Also this should be used only for test either way.

Reply:
> Recommend removing aliasing to avoid breaking post RC to GA. Also this should be used only for test either way.

Got it. Updated the code to remove aliasing.